src/System.Runtime.Extensions/src/System/IO/TextWriter.cs

```diff
@@ -109,7 +109,9 @@ public virtual string NewLine
         // but descendant classes can override the method to provide the
         // appropriate functionality.
         //
-        public abstract void Write(char value);
+        public virtual void Write(char value)
+        {
+        }
```

Comment:
As discussed offline - TextWriter has nothing to write to - all other Write() functions go through here (which incidentally is not too efficient if you're logging strings)

If I make it throw NotImplementedException it would make it obvious that one's TW derived class is incomplete but that will be obvious already to anyone using the existing Write methods as nothing will work. It would also break anyone with a perfectly good derived class that unnecessarily does base.Write.

So I"ll leave as is.